roadmap/src/main/webapp/question.html

```diff
@@ -19,63 +19,113 @@
     <meta name="google-signin-client_id" content="1055777818620-l4sjm83glehidqtf7niespm6qq5rjeig.apps.googleusercontent.com.apps.googleusercontent.com">
   </head>
 <body class="bg question-banner">
-  <div class="mask" style="background-color: rgba(0,0,0,0.6)">
+  <div class="mask" style="height: 100%; background-color: rgba(0,0,0,0.6); overflow:scroll">
   <div class="row">
     <div class="col-12">
-      <div class="col-6 offset-3 white-bg">
+      <div class="col-6 offset-3 white-bg" style="height:104%; filter:drop-shadow(0 0 0.75rem #000);">
         <div class="col-8 offset-2 d-flex justify-content-center">
           <div class="mx-auto">
             <a href="/#"><img src="img/node-tree.svg" width=90 height=90 alt="Node Tree"></a>
           </div>
         </div>
-        <form method="POST" action="/data" style="position:relative; top:55px;">
+        <form method="POST" class="row g-3 needs-validation" name="signupForm" action="/tree-landing.html" onSubmit="return checkInput(this)" style="position:relative; top:55px;" novalidate>
           <div class="col-md-8 offset-2">
-            <h3 style="margin-bottom:75px">Welcome, <span id="fname"></span></h3>
-            <div class="text-center">
+            <h3 style="margin-bottom:30px">Welcome, <span id="fname"></span></h3>
+            <hr>
+            <div class="text-center p-2">
                 <h4>Lets get to know you better!</h4>
             </div>
           </div>
-          <div class="form-group col-md-8 offset-2">
+          <div class="form-outline btn-rounded form-floating col-md-8 offset-2">
+            <input type="text" class="form-control" name="college" placeholder="College" required>
             <label class="form-label">What College Do You Attend?</label>
-            <input type="text" class="form-control" name="college" placeholder="College" 
-            required>
           </div>
-          <div class="form-group col-md-8 offset-2">
```

Comment:
I also think we keep on clutching on the way we want to present this form. I like the edits that i had made in that it made it look neater.

Reply:
How about I just expand the whole thing to give everything some space

See the image below and let me know what you'd like me to do

### Current:
<img width="1440" alt="Screen Shot 2021-04-12 at 4 39 45 PM" src="https://user-images.githubusercontent.com/51393498/114458853-c1eb4180-9bad-11eb-8bde-9468215d77ae.png">

### Adjusted:
<img width="1440" alt="Screen Shot 2021-04-12 at 4 39 35 PM" src="https://user-images.githubusercontent.com/51393498/114458928-db8c8900-9bad-11eb-970d-12aba2ea7697.png">

Reply:
I don't have a problem with that! I was mostly talking about the actual input and tag layout and the classes that go into. I realized that in working on the page, I changed your initial edits and you put them back.